Sources/MongoSwift/MongoCollection+BulkWrite.swift

```diff
@@ -316,7 +316,7 @@ public struct BulkWriteOptions: Codable {
 }
 
 /// The result of a bulk write operation on a `MongoCollection`.
-public struct BulkWriteResult: Decodable {
+public struct BulkWriteResult: Codable {
```

Comment:
I ended up just making all of these write result types Codable for the purpose of matching results. I don't see why it would be an issue since they all conform in a standard manner, and I highly doubt users will rely on the conformance anyway.